go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/denis-tingajkin/go-header v0.4.2
 	github.com/esimonov/ifshort v1.0.0
 	github.com/fatih/color v1.10.0
-	github.com/go-critic/go-critic v0.5.3
+	github.com/go-critic/go-critic v0.5.4-0.20210131194654-6090d88757f4
```

Comment:
Waiting for go-critic 0.5.4 to be released.

Reply:
https://github.com/go-critic/go-critic/releases/tag/v0.5.4